esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2245,6 +2245,46 @@ void WsWuInfo::getWUProcessLogSpecs(const char* processName, const char* logSpec
         logSpecs.sortAscii(false); //Sort the logSpecs from old to new
 }
 
+bool WsWuInfo::validateWUProcessLog(const char* file, bool eclAgent)
+{
+    Owned<IStringIterator> logs = cw->getLogs(eclAgent ? "EclAgent" : "Thor");
+    ForEach (*logs)
+    {
+        SCMStringBuffer logName;
+        logs->str(logName);
+        if (logName.length() < 1)
```

Comment:
question: is it worth checking this at all - won't it fail the strieq test anyway?